server/src/packages/packages.controller.ts

```diff
@@ -1,6 +1,8 @@
-import { Controller, Get, Param, UseInterceptors } from '@nestjs/common';
+import { Controller, Get, Param, UseInterceptors, Patch, Body, Session, HttpException, HttpStatus, UsePipes, UseGuards } from '@nestjs/common';
```

Comment:
V. nit:  when these lines get long it's more readable to break each variable onto a newline (code formatters like Prettier would do this automatically)

Reply:
@bfreeds  will do, can we add it to our lint rules?

Reply:
Tried getting this to work, might need some help: https://eslint.org/docs/rules/object-curly-newline